models/tracker/src/index.ts

```diff
@@ -576,6 +579,11 @@ export function createModel (builder: Builder): void {
         component: tracker.component.SetParentIssueActionPopup,
         element: 'top'
       },
+      query: {
+        // Hide this action if the issue has subtasks to avoid circular
+        // dependencies and assigning the issue to its own subtask.
+        subIssues: 0
```

Comment:
I'm not agree.

A -> {B, C} 
C-> D
D->E
and I want to change parent for C, i should be able to do so.